tests/models/fast/test_modeling_fast.py

```diff
@@ -0,0 +1,350 @@
+        self.model_tester.create_and_check_model(*config_and_inputs)
+
+    def create_and_test_config_common_properties(self):
+        return
+
+    @unittest.skip(reason="Fast does not use inputs_embeds")
+    def test_inputs_embeds(self):
+        pass
+
+    @unittest.skip(reason="Fast does not support input and output embeddings")
+    def test_model_common_attributes(self):
+        pass
+
+    @unittest.skip(reason="Fast is not a generative model")
+    def test_generate_without_input_ids(self):
+        pass
+
+    @unittest.skip(reason="Fast is does not have any hidden_states")
+    def test_hidden_states_output(self):
+        pass
+
+    @unittest.skip(reason="Fast is does not have any attention")
+    def test_retain_grad_hidden_states_attentions(self):
+        pass
+
+    def test_forward_signature(self):
+        config, _ = self.model_tester.prepare_config_and_inputs_for_common()
+
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            signature = inspect.signature(model.forward)
+            # signature.parameters is an OrderedDict => so arg_names order is deterministic
+            arg_names = [*signature.parameters.keys()]
+
+            expected_arg_names = ["pixel_values"]
+            self.assertListEqual(arg_names[:1], expected_arg_names)
+
+    def _prepare_for_class(self, inputs_dict, model_class, return_labels=False):
+        to_return = inputs_dict.copy()
+        gt_instances = torch.zeros(
+            self.model_tester.batch_size, self.model_tester.image_size, self.model_tester.image_size
+        )
+        gt_kernels = torch.zeros(
+            self.model_tester.batch_size, self.model_tester.image_size, self.model_tester.image_size
+        )
+        gt_text = torch.zeros(self.model_tester.batch_size, self.model_tester.image_size, self.model_tester.image_size)
+        training_masks = torch.ones(
+            self.model_tester.batch_size, self.model_tester.image_size, self.model_tester.image_size
+        )
+        labels = {}
+        labels["gt_instances"] = gt_instances
+        labels["gt_kernels"] = gt_kernels
+        labels["gt_texts"] = gt_text
+        labels["training_masks"] = training_masks
+
+        to_return["labels"] = labels
+
+        return to_return
+
+    @unittest.skip(reason="FAST params are bigger than 80M params")
+    def test_model_is_small(self):
+        pass
+
+    @unittest.skip(reason="FAST does not have input/output embeddings")
+    def test_model_get_set_embeddings(self):
+        pass
+
+    @unittest.skip(reason="Fast does not support input and output embeddings")
+    def test_generate_from_inputs_embeds(self):
+        pass
+
+    @unittest.skip(reason="Fast does not generate text")
+    def test_generation_tester_mixin_inheritance(self):
+        pass
+
+
+@require_vision
+class FastModelIntegrationTest(unittest.TestCase):
+    # @slow
+    def test_inference(self):
```

Comment:
use `$ RUN_SLOW=1 pytest ...` to run such tests

```suggestion
    @slow
    def test_inference(self):
```